### DISCUSSION TITLE
ScriptReference/Il2CppCompilerConfiguration

### DISCUSSION THREAD

**Discussion by docscomments** [Developer-Notes-Extension/Unity-Notes#384] [Notes]
# ScriptReference/Il2CppCompilerConfiguration

https://docs.unity3d.com/ScriptReference/Il2CppCompilerConfiguration.html

**Comment by rthery** [comment]
You can get/set this value via [`PlayerSettings.GetIl2CppCompilerConfiguration`](https://docs.unity3d.com/ScriptReference/PlayerSettings.GetIl2CppCompilerConfiguration.html) and [`PlayerSettings.SetIl2CppCompilerConfiguration`](https://docs.unity3d.com/ScriptReference/PlayerSettings.SetIl2CppCompilerConfiguration.html)

**Comment by abrusle** [comment]
I haven't tested this, but selecting `Master` should also strip out profile support.
*Source: https://stackoverflow.com/a/20218234/8285600*
